src/gui/DatabaseWidget.cpp

```diff
@@ -984,7 +984,10 @@ void DatabaseWidget::unlockDatabase(bool accepted)
     }
     replaceDatabase(db);
     if (db->isReadOnly()) {
-        showMessage(tr("File opened in read only mode."), MessageWidget::Warning, false, -1);
+        showMessage(tr("This database is opened in read only mode. Auto-save has been disabled."),
```

Comment:
"read-only" and "Autosave".

Perhaps also "is disabled" instead of "has been disabled"?